code/modules/recycling/sortingmachinery.dm

```diff
@@ -273,8 +273,7 @@
 	icon_state = "intake"
 	required_mode_to_deconstruct = 1
 	deconstructs_to = PIPE_DISPOSALS_CHUTE
-
-	var/c_mode = 0
+	var/c_mode = 0 //1 = we can deconstruct, 0 means we cannot

```

Comment:
Maybe rename the var to make more sense? c_mode doesn't tell a reader much.

Reply:
Might as well even though I was just moving shit round 😄